ensembl-track-api.openapi.yaml

```diff
@@ -89,4 +113,24 @@ components:
           example: GREY
         additional_info:
           type: string
-          example: Forward strand
\ No newline at end of file
+          example: Forward strand
+        description:
+          type: string
+          example: Shows all protein coding genes on the forward strand.\nPart of the GENCODE Comprehensive gene set
+        sources:
+          type: array
+          items:
+            type: object
+            properties:
+              name:
+                type: string
+                example: GENCODE
+              url:
+                type: string
+                example: https://gencodegenes.org
+        rank:
+          type: number
+          example: 2
```

Comment:
are we going to start counting from 0 or from 1?

Reply:
Maybe more descriptive would be `order` or `display_order`?

Reply:
I don't mind `display_order` 🙂 On the other hand, the proposal for the search api marks the order of the matches with a `rank` field. And regions in CDM are going to have `rank` ([link](https://github.com/Ensembl/ensembl-cdm-docs/pull/58/files#diff-7ffc5ffdf4eea8d7fb53c043adf7e23d8a1ff0b9a5df4b79c25f0ec33883f4e1)). Does this matter?